eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -184,6 +184,25 @@ jobs:
         BuildTargetingString: ${{ parameters.BuildTargetingString }}
         TestMarkArgument: ${{ parameters.TestMarkArgument }}
 
+  - job: 'RunApiStubGen'
+    condition: and(succeededOrFailed(), ne(variables['Skip.ApuStubGen'], 'true'))
```

Comment:
"ApuStubGen" -> "ApiStubGen"

Reply:
Oh no. I am fixing it. Thanks for the review.